frontend/src/pages/GraphicalSelector/CourseInfo/CourseInfo.tsx

```diff
@@ -0,0 +1,165 @@
+          unlocked: results[2].data,
+        });
+      } catch (e) {
+        // eslint-disable-next-line no-console
+        console.error('Error at getCourse', e);
+      }
+    };
+    getInfo();
+  }, [courseCode, degree, planner]);
+
+  if (!info) {
+    return (
+      <S.Wrapper>
+        <LoadingCourseInfo />
+      </S.Wrapper>
+    );
+  }
+
+  const { course, pathFrom, unlocked } = info;
+
+  return (
+    <S.Wrapper>
+      <Title level={2} className="text">{courseCode} - {course.title}</Title>
+      <PlannerButtonCode course={course} onChange={onCourseAdd} />
+      <S.TermWrapper>
+        <Text strong>Terms: </Text>
+        {course.terms.length
+          ? course.terms.map((term) => (
+            <TermTag key={term} name={term === 'T0' ? 'Summer' : `Term ${term.slice(1)}`} />
+          ))
+          : 'None'}
+      </S.TermWrapper>
+      <S.MiscInfo>
+        <S.MiscInfoChild>
+          <Text>{course.study_level}</Text>
+          <br />
+          <Text>{course.campus}</Text>
+        </S.MiscInfoChild>
+        <S.MiscInfoChild>
+          <Text>{course.school}</Text>
+        </S.MiscInfoChild>
+        <S.MiscInfoChild>
+          <Text strong>{course.UOC} UOC</Text>
+          <br />
+          <Text>View Handbook</Text>
+        </S.MiscInfoChild>
+      </S.MiscInfo>
+      <Collapsible title="Overview">
+        <p>{course.description}</p>
+      </Collapsible>
+      <Collapsible title="Requirements" initiallyCollapsed>
+        <p>{course.raw_requirements}</p>
+      </Collapsible>
+      <Collapsible title="Courses you have done to unlock this course" initiallyCollapsed>
+        <p>
+          {pathFrom && pathFrom.length > 0 ? (
+            pathFrom
+              .filter((code) => Object.keys(planner.courses).includes(code))
+              .map((code) => (
+                onCourseClick
+                  ? (
+                    <GraphicalCourseTag
+                      key={code}
+                      name={code}
+                      onClick={() => { onCourseClick(code); }}
+                    />
+                  )
+                  : <GraphicalCourseTag key={code} name={code} />
+              ))
+          ) : 'None'}
+        </p>
+      </Collapsible>
+      <Collapsible title="Doing this course will directly unlock these courses" initiallyCollapsed>
+        <p>
+          {unlocked.direct_unlock && unlocked.direct_unlock.length > 0 ? (
+            unlocked.direct_unlock.map((code) => (
+              onCourseClick
+                ? (
+                  <GraphicalCourseTag
+                    key={code}
+                    name={code}
+                    onClick={() => { onCourseClick(code); }}
+                  />
+                )
+                : <GraphicalCourseTag key={code} name={code} />
+            ))
+          ) : 'None'}
+        </p>
+      </Collapsible>
+      <Collapsible title="Doing this course will indirectly unlock these courses" initiallyCollapsed>
```

Comment:
This copy pastes from the original components from the courseSelector. We should factor the commonalities into shared components

Reply:
If there are small changes, we can call one of the views a "compact view"